addons/medical_status/CfgEventHandlers.hpp

```diff
@@ -19,6 +19,7 @@ class Extended_PostInit_EventHandlers {
 class Extended_Init_EventHandlers {
     class CAManBase {
         class ADDON {
+            onRespawn = true;
```

Comment:
This only accepts number 1, no strings. Booleans don't exist in config.
https://github.com/CBATeam/CBA_A3/blob/master/addons/xeh/fnc_compileEventHandlers.sqf#L155

Reply:
Sorry about that. Took it from the XEH wiki and didn't notice the define statement 4 lines above it... Should be fixed now.

Reply:
Just looked in the config viewer as I was wondering why this mistake didn't cause unwanted behavior and noticed that it already got converted from true to 1.  I'll leave the change in for clarity.

Reply:
Not all binarizers do this. Mikero converts it despite no define, which causes a lot of other problems.